src/trackByFunctionRule.ts

```diff
@@ -24,28 +24,26 @@ export class Rule extends Lint.Rules.AbstractRule {
   }
 }
 
+const ngForExpressionRe = new RegExp(/\*ngFor="(.+)"/);
```

Comment:
`/\*ngFor\s*=\s*'|"(.+)'|"/` this might be more accurate it will catch`*ngFor = 'foobar'` as well.

Reply:
@mgechev Yep, good point. Will add that in. Also did you see my comment above about node-sass?

Reply:
`node-sass` is fine, we use it only in the tests.

Reply:
Would you also add a test case with different quotes and whitespace?

Reply:
Ok, added a couple more. Should cover all the cases in the Regex.